libmscore/read206.h

```diff
@@ -68,6 +68,7 @@ class PageFormat {
 extern Element* readArticulation(Score*, XmlReader&);
 extern void readAccidental206(Accidental*, XmlReader&);
 extern void setPageFormat(MStyle*, const PageFormat&);
+extern void setPageFormat114(MStyle*, const PageFormat&);
```

Comment:
A small oversight:
setPageFormat114 should be merged with setPageFormat - only one function is required now.  I'll work on it...